src/source-updater.js

```diff
@@ -170,7 +170,11 @@ const actions = {
     }
 
     sourceUpdater.logger_(`Removing ${start} to ${end} from ${type}Buffer`);
-    sourceBuffer.remove(start, end);
+    try {
+      sourceBuffer.remove(start, end);
```

Comment:
Apparently newer versions of chrome can fail to remove on a source buffer if there isn't anything to remove.